src/ts/utils/prosemirror.ts

```diff
@@ -122,45 +143,53 @@ export const getReplacementFragmentsFromReplacement = (
           currentPos: currentPos + patch.count
         };
       }
-      const $from = tr.doc.resolve(from + currentPos);
-      const $to = tr.doc.resolve(Math.min($from.pos + patch.count, tr.doc.nodeSize - 2));
+
+      const newFrom = from + currentPos;
+      const newTo = newFrom + patch.count;
 
       // If this patch removes chars, create a fragment for
       // the range, and leave the cursor where it is.
       if (patch.removed) {
+        const fragment = {
```

Comment:
```suggestion
        const fragment : : ISuggestionPatchDelete = {
```
`nit` Is this a nicer way to create an instance, as it means you don't need the `as const`?